dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'TZ': [Tanzânia, Tanzania, Tanzanian*, Dodoma, Dar es Salaam, Dar es Salã]
+    'UG': [Uganda, Ugandense*, Ugandês*, Campala]
+    'YT': [Mayotte, Maiote, Maiotense*, Mamudzu]
+    'ZM': [Zâmbia, Zambian*, Lusaca]
+    'ZW': [Zimbabwe, Zimbábue, Zimbabué, Zimbaué, Zimbabuean*, Zimbabwean*, Zimbabuan*, Zimbabwan*, Zimbabuense*, Harare]
+
+  MIDDLE:
+    'AO': [Angola, Angolan*, Angolense*, Luanda]
+    'CD': [República Democrática do Congo, RDC, Congo, Congo-Quinxassa, Congo-Quinxasa, Congolês*, Congolense*, Conguês*, Quinxassa, Quinxasa, Kinshasa]
+    'CF': [República Centro-Africana, República da África-Central, Centro-African*, Bangui]
+    'CG': [República do Congo, Congo-Brazavile, Congo-Brazzaville , Congolês*, Conguês*, Conguito*, Congolense*, Brazavile, Brazzaville]
+    'CM': [Camarões, Cameronense*, Cameronês*, Iaundê, Iaundé]
+    'GA': [Gabão, República Gabonesa, Gabonense*, Gabonês*, Libreville, Librevile]
+    'GQ': [Guiné Equatorial, Guineense*, Guinéu-Equatorian*, Equato-Guineense*, Malabo]
+    'ST': [São Tomé e Príncipe, Santomense*, São-Tomense*, São Tomé]
+    'TD': [Chade, Tchade, Chadian*, Chadiense*, Jamena, N'Djamena]
+
+  NORTH:
+    'DZ': [Argélia, República Argelina Democrática e Popular, Argelin*, Argel]
+    'EG': [Egito, Egípci*, Egipcian*, Egipcíac*, Cairo]
+    'EH': [Saara Ocidental, Sara Ocidental, Sáara Ocidental, Saaráui*, Saarian*, Sarian*, El Aiune, Laiune]
+    'LY': [Líbia, Líbio*, Líbic*, Trípoli]
+    'MA': [Marrocos, Marroquin*, Rabat, Rabate]
+    'SD': [Sudão, Sudanês*, Cartum]
+    'SS': [Sudão do Sul, Sudão Meridional, Sul-Sudanês*, Juba]
+    'TN': [Tunísia, República Tunisina, Tunisian*, Tunisin*, Tunes, Túnis]
+
+  SOUTH:
+    'BW': [Botswana, Botsuana, Bechuan*, Botsuanense*, Botsuan*, Botsuanês*, Gaborone]
+    'LS': [Lesoto, Lessoto, Lesotian*, Lestotense*, Lesoto*, Maseru]
+    'NA': [Namíbia, Namibian*, Vinduque]
+    'SZ': [Essuatíni, Essuatinian*, Lobamba, Mebabane, Mbabane]
+    'ZA': [África do Sul, Sul-African*, Austro-African*, Cidade do Cabo, Joanesburgo, Johannesburgo, Pretória]
+
+  WEST:
+    'BF': [Burquina Fasso, Burquina Faso, Burquina, Burquinense*, Burquinabê, Burquinês*, Burquin*, Uagadugu]
+    'BJ': [Benim, Beninês*, Beninense*, Porto Novo]
+    'CI': [Costa do Marfim, República de Côte d'Ivoire, Costa-Marfinense*, Marfinense*, Ebúrne*, Ivoiriense*, Iamussucro, Abidjã, Abijã, Abijão, Abidjan]
+    'CV': [Cabo Verde, Cabo-Verdian*, Praia]
+    'GH': [Gana, Ganês*, Ganense*, Acra]
+    'GM': [Gâmbia, Gambian*, Banjul]
+    'GN': [Guiné, Guinean*, Guineense*, Conacri]
+    'GW': [Guiné-Bissau, Guineense*, Bissau]
+    'LR': [Libéria, Liberian*, Monróvia]
+    'ML': [Mali, Máli, Malian*, Malinês*, Malês*, Bamaco]
+    'MR': [Mauritânia, Mauritan*, Nuaquexote, Nuaquechote]
+    'NE': [Níger, Nigerin*, Nigerenseé*, Niamei, Niamey]
+    'NG': [Nigéria, Nigerian*, Abuja, Lagos]
+    'SH': [Santa Helena, Santo, Santa-Helenense*, Jamestown]
```

Comment:
@bah-elly Not sure why Santo is included. Is it the demonym?

Reply:
Yes, it is the demonyn. I was having trouble finding a demonyn for this case and when googling the island this was the main one that I could find being used to reference the population.